docker/templates/Dockerfile.j2

```diff
@@ -29,7 +29,7 @@ FROM {{ base_image }}
 # Install Java and the "which" command, which is needed by Logstash's shell
 # scripts.
 # Minimal distributions also require findutils tar gzip (procps for integration tests)
-RUN {{ package_manager }} update -y && {{ package_manager }} install -y procps findutils tar gzip java-11-openjdk-devel which && \
+RUN {{ package_manager }} update -y && {{ package_manager }} install -y procps findutils tar gzip which shadow-utils && \
```

Comment:
@robbavey shadow-utils bring in `groupadd` used at line 36, probably was a transitive dependency of the `java-11-openjdk-devel`

Reply:
Good catch